src/rp2_common/hardware_flash/include/hardware/flash.h

```diff
@@ -121,6 +121,132 @@ void flash_do_cmd(const uint8_t *txbuf, uint8_t *rxbuf, size_t count);
 
 void flash_flush_cache(void);
 
+#if !PICO_RP2040
```

Comment:
ughh.. this file is using TABs not 4 spaces

Reply:
oh, yeah. My editor is set up to guess from the file, but there were no indented lines in here.